src/test/fuzz/feefrac.cpp

```diff
@@ -217,5 +218,16 @@ FUZZ_TARGET(feefrac_mul_div)
             FeeFrac{mul64, div}.EvaluateFeeDown(mul32) :
             FeeFrac{mul64, div}.EvaluateFeeUp(mul32);
         assert(res == res_fee);
+
+        // Compare approximately with CFeeRate.
+        if (mul64 <= std::numeric_limits<int64_t>::max() / 1000 &&
+            mul64 >= std::numeric_limits<int64_t>::min() / 1000 &&
+            quot_abs <= arith_uint256{std::numeric_limits<int64_t>::max() / 1000}) {
+            CFeeRate feerate(mul64, (uint32_t)div);
+            CAmount feerate_fee{feerate.GetFee(mul32)};
+            auto allowed_gap = static_cast<int64_t>(mul32 / 1000 + 3 + round_down);
```

Comment:
In "fuzz: assert min diff between FeeFrac and CFeeRate" how did you come up with 1000 and 3 here?
I could not comprehend the computation here. how did you come up with this gap.